frontend/views/reports.js

```diff
@@ -85,6 +133,7 @@ module.exports = Backbone.View.extend({
         this.$el.html(template(context));
         this.resultsView = new ReportTableView({
```

Comment:
Oh!  I should look at the whole file and not just the diff. There are two different view "classes" but I only saw one.